pkg/controller/druid/handler.go

```diff
@@ -174,7 +174,11 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 				if !done {
 					return err
 				}
+
 			}
+
+			checkCrashStatus(sdk, m)
```

Comment:
can we hide this feature behind a flag and default to true?

also, for rolling deploy enabled case, this needs to happen before line 173.

Reply:
sure, i had that doubt if this should be flagged, will do that.